src/components/common/animations/Error/index.js

```diff
@@ -19,7 +19,8 @@ class Error extends AnimationBase {
         style={{
           width: getDesignRelativeWidth(97, false),
           marginHorizontal: 'auto',
-          marginVertical: 'auto',
+          paddingTop: 20,
+          paddingBottom: 20,
```

Comment:
These styles have been fixed to work cross-platform.
conserve marginVertical:'auto' please.